todo-app/src/Stores/index.js

```diff
@@ -0,0 +1,9 @@
+import { configure } from 'mobx'
+
+import TasksListStore from './TasksListStore'
+
+configure({ enforceActions: true })
+
+export default {
+    data: TasksListStore,

```

Comment:
You shouldn't be calling your store `data`... Imagine we had a few other stores. You can name the key here exactly the same as the store if you like, or use camel case. Up to you.